website/app/controllers/application.js

```diff
@@ -1,15 +1,44 @@
 import Controller from '@ember/controller';
 import { action } from '@ember/object';
 import { tracked } from '@glimmer/tracking';
+import { scheduleOnce } from '@ember/runloop';
+import { inject as service } from '@ember/service';
 
 export default class ApplicationController extends Controller {
-  @tracked showSidebarOnSmallViewport = false;
+  @service router;
+  @service fastboot;
 
-  // TODO add a reset for the `showSidebarOnSmallViewport` on route change
+  // notice: don't set it to false, otherwise the sidebar menu will appear open on page load
+  @tracked isSidebarVisibleOnSmallViewport;
+
+  constructor() {
+    super(...arguments);
+    this.router.on('routeDidChange', this, 'routeDidChange');
+  }
+
+  routeDidChange() {
+    scheduleOnce('afterRender', this, this.onToggleBurgerMenu);
+  }
+
+  willDestroy() {
+    this.router.off('routeDidChange', this, 'routeDidChange');
+  }
+
+  get currentTopRoute() {
```

Comment:
it doesn't look like this behavior is working any longer. if you navigate to `/components` the class `is-current-route` isn't present where it is in production.

It may be possible to rework this logic to leverage the `current-when` behavior i was investigating yesterday but i'm not sure

Reply:
fixed in https://github.com/hashicorp/design-system/pull/1009/commits/3bf4b96c9fb11a8cca40a82e00be735cb4db198b